.github/workflows/test.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         path: slam
 
     - name: make buf
-      timeout-minutes: 5
+      timeout-minutes: 10
```

Comment:
Increasing the timeouts for `make buf` while I'm here, since this sometimes times out